database/postgresqlDB.js

```diff
@@ -8,15 +8,129 @@ const pool = new Pool({
 });
 
 // test query
-pool.query('SELECT * FROM time_slots T', (error, results) => {
-  if (error) {
-    throw error;
-  } else {
-    console.log((results.rows));
-    pool.end();
-  }
-});
+// pool.query('SELECT * FROM time_slots T', (error, results) => {
+//   if (error) {
```

Comment:
Same as other comments - if you're keeping this around, make a note as to why you've commented it out, what it does, and why you're keeping it around.